### PR TITLE
Enter test result

### PR DESCRIPTION
**Pull request by abinaya-u**
## Requirements

- [x] This PR has a proper title that briefly describes the work done
- [x] I have squashed / amended the comments to make it more redable
- [x] I have included link to all the JIRA ticket('s)
- [x] I wrote the code as a pair or atleast performed a self-review of my own code
- [x] I have updated the documentation on [Bahmni Wiki](https://bahmni.atlassian.net/wiki/spaces/BAH/overview) or [README](https://github.com/Bahmni/bahmni-lab-frontend/blob/main/README.md) (if applicable)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Summary
User should be able to enter test results for the tests from the open orders. This PR has changes to enter tests results for stand alone tests and **not for panel**. After successful save, user should able to see the value in bahmni's clinical dashboard. User can also mark a entered value as abnormal which will highlighted in red colour and on successful save, user will be able to see the entered value in red colour in clinical dashboard.

## Screenshots
image
image

## JIRA tickets
https://bahmni.atlassian.net/browse/BAH-2998
https://bahmni.atlassian.net/browse/BAH-3041
https://bahmni.atlassian.net/browse/BAH-3044
https://bahmni.atlassian.net/browse/BAH-3055

### CODE REVIEW

**Comment by gsluthra** [comment]
Plz ensure all committers have signed CLA. Else we shouldn't merge this PR. Code wise, I don't have comments, except to say --- Big PR! : )

**Comment by gokultw** [comment]
@pankaj2961 @JagadeeshM07 Please sign CLA.

**Comment by abinaya-u** [comment]
> Plz ensure all committers have signed CLA. Else we shouldn't merge this PR. Code wise, I don't have comments, except to say --- Big PR! : )

all the committers have signed CLA. Can you please review this PR?

**Review by gsluthra — APPROVED** [review]

**Pull request merged**
